### PR TITLE
Janggi arena points

### PR DESCRIPTION
**Pull request by gbtami**

### CODE REVIEW

File: server/tournament.py [hunk 1]
```diff
@@ -697,6 +689,79 @@ def points_perfs(self, game):
...
+
+        if game.result == "1-0":
+            wplayer.nb_win += 1
+
+            if self.system == ARENA:
+                if wplayer.win_streak == 2:
+                    if game.status == VARIANTEND:
+                        bpoint = (2, SCORE)
+                        wpoint = (4 * 2, SCORE)
+                        wplayer.win_streak = 0
```
Comment (Fulmene) [inline server/tournament.py:709]: You'll need to cover the case where the loser is on a win streak too.

**Review by Fulmene — CHANGES_REQUESTED** [review]

File: server/tournament.py (hunk 1 above)
Comment (gbtami) [reply server/tournament.py]: Thx, fixed it.

**Comment by Fulmene** [comment]
If we treat material counting as a draw, then should the "10 moves or more" rule apply too?

Actually, I think the code might look better if you just check for `if game.status == VARIANTEND` first, then the points_perfs_janggi function will look the exact same way as regular points_perfs. Like so:

```
if game.status == VARIANTEND:
    # Do some logic for drawn games
    if game.result == "1-0":
        # Cho won by material count
    else:
        # Han won by material count
elif: game.result == "1-0":
    # Cho won by checkmate
elif game.result == "0-1":
    # Han won by checkmate
```

**Comment by gbtami** [comment]
Updated according to https://discord.com/channels/634298688663191582/641327060090093581/911216993368158308

**Review by Fulmene — APPROVED** [review]

**Comment by gbtami** [comment]
Thx for the review!

**Pull request merged**